include/dba.php

```diff
@@ -91,6 +91,22 @@ public function getdb() {
 		return $this->db;
 	}
 
+	/** Returns the MySQL server version string
```

Comment:
According to the Doxygen docs, the @brief description can be set on the opening tag line without the "@brief" mention. Is it another implicit convention for documentation?